draft-patton-cfrg-vdaf.md

```diff
@@ -861,15 +865,21 @@ pseudorandom field elements.
 ~~~
 # Derive a new seed.
 def derive(Prg, seed: bytes, info: bytes) -> bytes:
-    return Prg.expand(seed, info, Prg.SEED_SIZE)
+    prg = Prg(seed, info)
+    return prg.next(Prg.SEED_SIZE)
 
 # Expand a seed into a vector of Field elements.
```

Comment:
Same as above, please make sure that this is always used in a safe way by requiring that the seed MUST have high entropy and be only used once.

Reply:
It depends on the PRG construction, but it should be secure to reuse the same seed with different "info" strings. That is, the intent of the "info" string is to provide domain separation.

Here I stipulated the same requirements for generating the seed as above.

Reply:
Okay, let's maybe add a short sentence on this to the documentation (see comment above).